packages/dds/tree/src/feature-libraries/fence.json

```diff
@@ -3,5 +3,5 @@
     "exports": [ "index"],
     // Depending on "shared-tree-core" so implementations of Indexes can be here.
     // Could move the index abstraction elsewhere instead.
-    "imports": ["schema-stored","schema-view","tree","util","rebase","forest","dependency-tracking","shared-tree-core","change-family"]
+    "imports": ["schema-stored","schema-view","tree","util","rebase","forest","dependency-tracking","shared-tree-core","change-family","domains","shared-tree","checkout"]
```

Comment:
We should not take a dependency on shared-tree here. feature libraries are depended on by shared tree, so this creates a cycle which we want to avoid. See the architectural diagrams in the package readme.